### ISSUE TITLE
Not ablw to start my reactnative project after install this package 

### ISSUE THREAD

**Issue by Virajmakwana** [NodeMedia/react-native-nodemediaclient#177]
> Could not create task ':react-native-nodemediaclient:compileDebugJavaWithJavac'. showing this error

**Comment by helyz007** [comment]
I have the same issue when after updating my expo managed to expo 49. Do you later have any solution for the problem?

**Comment by Virajmakwana** [comment]
> I have the same issue when after updating my expo managed to expo 49. Do you later have any solution for the problem?

I am facing this issue for the latest version of the package in the version 0.2.18 its working fine

**Comment by helyz007** [comment]
@Virajmakwana The issue still persist on expo managed (Expo 49) even after changing the version to 0.2.18

**Comment by helyz007** [comment]
![Screenshot_20230918-211726](https://github.com/NodeMedia/react-native-nodemediaclient/assets/78934468/2cbd38f5-b3d3-43ad-9529-df8640d8bbf0)
This is the error I am getting whenever I used eas build

**Comment by Virajmakwana** [comment]
yes i was also facing same issue but when i used version 0.2.18, i didn't get it

**Comment by Veryinheart** [comment]
I got same error when build locally for Android. I tried downgrade to 0.2.18 but still got same error. which RN version do you use? @Virajmakwana
